topi/tests/python/test_topi_pooling.py

```diff
@@ -294,6 +285,11 @@ def test_adaptive_pool():
     verify_adaptive_pool((1, 3, 224, 224), (1, 1), "avg")
     verify_adaptive_pool((1, 14, 56, 78), (34, 13), "max")
     verify_adaptive_pool((1, 5, 46, 97), (4, 96), "avg")
+    verify_adaptive_pool((1, 16, 32, 32, 32), (1, 1, 1), "max", layout="NCDHW")
+    verify_adaptive_pool((1, 16, 32, 32, 32), (1, 1, 1), "avg", layout="NCDHW")
+    verify_adaptive_pool((1, 16, 32, 32, 32), (2, 2, 2), "max", layout="NCDHW")
+    verify_adaptive_pool((1, 16, 32, 32, 32), (2, 2, 2), "avg", layout="NCDHW")
```

Comment:
We should probably include one test with a more complex output shape, one NDHWC layout test would be great too.

Reply:
Added in the commit https://github.com/apache/incubator-tvm/pull/5049/commits/90c22195ec323f7606fb8c8c52f1b5bb45ab0313. 

I found that other topi pool tests don't test on NHWC layout at all. Tests and numpy reference are all hard coded for NCHW.